webpack.prod.config.js

```diff
@@ -13,15 +13,12 @@ const build = process.env.BUILD_NUMBER || 'SNAPSHOT'
 const config = require('./src/config.json')
 
 // const ethereumHost = process.env.ETHEREUM_HOST
-const gnosisDbUrl = process.env.GNOSISDB_HOST || `${config.gnosisdb.protocol}://${config.gnosisdb.host}:${config.gnosisdb.port}`
+const gnosisDbUrl =
+  process.env.GNOSISDB_HOST || `${config.gnosisdb.protocol}://${config.gnosisdb.host}:${config.gnosisdb.port}`
 
 module.exports = {
   context: path.join(__dirname, 'src'),
-  entry: [
-    'bootstrap-loader/extractStyles',
```

Comment:
Why not extractStyles for prod?

Reply:
@andre-meyer extractStyles can be specified by two ways:
1. with entry like this
2. Add a parameter in .bootstraprc
We had it specified both ways and to avoid duplication I decided to remove it from config and leave it in .bootstraprc